ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplDumpTask.java

```diff
@@ -166,7 +166,9 @@ private void dumpEvent(NotificationEvent ev, Path evRoot, Path cmRoot) throws Ex
   }
 
   private ReplicationSpec getNewEventOnlyReplicationSpec(Long eventId) throws SemanticException {
-    return getNewReplicationSpec(eventId.toString(), eventId.toString());
+    ReplicationSpec rspec = getNewReplicationSpec(eventId.toString(), eventId.toString());
+    rspec.setIsEventDump(true);
```

Comment:
may be then we use the same terminology everywhere as during most of replication conversations we talk in bootstrap dump and incremental dump(not event dump)  :) .